cli/example_plugins/go_highlight/.gitignore

```diff
@@ -0,0 +1,2 @@
+/go-highlight-bb-plugin
```

Comment:
should we include the hash in the filename and store this in the user temp directory under a plugins subdir?

Reply:
Discussed offline -- the CLI can provide a PLUGIN_CACHE_DIR where plugins can save build artifacts like this, so that we don't need this .gitignore stuff.

So `init.sh` would output the binary and SHA to `$PLUGIN_CACHE_DIR/{go-highlight-plugin,plugin_deps.sum}` and then `handle_bazel_output.sh` can expect to find those there.